mkosi/installer/moss.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+from collections.abc import Sequence
+
+from mkosi.bubblewrap import apivfs_cmd, bwrap
+from mkosi.context import Context
+from mkosi.types import PathString
+from mkosi.util import sort_packages
+
+
+def moss_cmd(context: Context) -> list[PathString]:
+    cmdline: list[PathString] = [
+        "moss",
+        "--directory",
+        f"{context.root}",
+        "--yes-all",
+    ]
+
+    return cmdline
+
+
+def setup_moss(
+    context: Context,
+) -> None:
+    (context.pkgmngr / "etc/moss/repo.d").mkdir(exist_ok=True, parents=True)
+    bwrap(context,
+        moss_cmd(context)
+        + ["repo", "add", "volatile", "https://dev.serpentos.com/volatile/x86_64/stone.index"],
+        network=True, env=context.config.environment,
+    )
```

Comment:
```suggestion
    bwrap(
        context,
        moss_cmd(context) + ["repo", "add", "volatile", "https://dev.serpentos.com/volatile/x86_64/stone.index"],
        network=True, 
        env=context.config.environment,
    )
```
the hardcoded url should probably be configurable via the mirror, which is usually done in the distribution module.

Reply:
We shouldn't use the `moss` command to add repositories. Instead, we should write the file ourselves to the right location (I hope a human readable format is used).